packages/pds/src/sequencer/sequencer-leader.ts

```diff
@@ -111,38 +114,42 @@ export class SequencerLeader {
   }
 
   async sequenceOutgoing() {
-    const unsequenced = await this.getUnsequenced()
-    const chunks = chunkArray(unsequenced, 2000)
-    for (const chunk of chunks) {
-      await this.db.transaction(async (dbTxn) => {
-        await Promise.all(
-          chunk.map(async (row) => {
-            await dbTxn.db
-              .updateTable('repo_seq')
-              .set({ seq: this.nextSeqVal() })
-              .where('id', '=', row.id)
-              .execute()
-            await this.db.notify('outgoing_repo_seq')
-          }),
-        )
-      })
-      await this.db.notify('outgoing_repo_seq')
-    }
+    await this.db.db
+      .updateTable('repo_seq')
+      .from((qb) =>
+        qb
+          .selectFrom('repo_seq')
+          .select([
+            'id as update_id',
+            sql<number>`nextval(${sql.literal(REPO_SEQ_SEQUENCE)})`.as(
+              'update_seq',
+            ),
+          ])
+          .where('seq', 'is', null)
+          .orderBy('id', 'asc')
+          .as('update'),
+      )
+      .set({ seq: sql`update_seq::bigint` })
```

Comment:
I have a feeling we still have to make an exception for sqlite here (probably wont like the `::bigint`).